doc/sphinx/language/gallina-extensions.rst

```diff
@@ -805,6 +805,45 @@ Section :ref:`gallina-definitions`).
    Most commands, like :cmd:`Hint`, :cmd:`Notation`, option management, … which
    appear inside a section are canceled when the section is closed.
 
+.. cmd:: Variable @ident : @type
```

Comment:
I suggest you reword the introduction of this section ("The sectioning mechanism can be used to to organize a proof in structured sections. Then local declarations become available (see Section Definitions).") and in particular that you remove the reference to the section Definition of the Gallina chapter.